src/workloads/tpch/denormalized/1/Q21.yml

```diff
@@ -0,0 +1,20 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run TPC-H query 21 against the denormalized schema for scale 1.
+
+Clients:
+  Default:
+    QueryOptions:
+      socketTimeoutMS: -1
+
+Actors:
+- Name: TPCHDenormalizedQuery21
+  Type: RunCommand
+  Database: tpch
+  Phases:
+  - LoadConfig:
+      Path: ../../../../phases/tpch/denormalized/Q21.yml
+      Key: TPCHDenormalizedQuery21
+      Parameters:
+        Query21Nation: "SAUDI ARABIA"
```

Comment:
That is a really good catch; I totally missed that! Fixed- it should now be `Query21Delta`. I believe it worked because the default is the same as the parameter value set in the workload definition.